integrations/docker/images/chip-build-cirque/README.md

```diff
@@ -0,0 +1,35 @@
+# Docker connectedhomeip:chip-build
```

Comment:
This documentation doesn't seem chip-build-cirque-specific.  Was it supposed to go in some ancestor dir?

Reply:
copy-pasta'd from chip-build/Dockerfile